vendor/jsdiff/package.json

```diff
@@ -2,5 +2,6 @@
   "name": "jsdiff",
   "private": true,
   "version": "1.0.0",
-  "main": "jsdiff.js"
+  "main": "jsdiff.js",
+  "types": "jsdiff.d.ts"
```

Comment:
For both of these `vendor/` changes, I've moved the associated `.d.ts` file into the package's folder and defined the `"types"` key in the `package.json.  This allows use to provide types for a vendored package without them being in our shared `types/` folder (eventually, it'd be nice to get rid of that as it's a bandaid for packages that don't ship types and is symlinked to all package folders).